examples/ex01_basics/IncrMethodModular_test.py

```diff
@@ -60,8 +60,8 @@ def construct( s ):
     s.buf2  = Buffer()
 
     # Connect the callee ports to buf1/buf2 write/read method ports
-    s.connect( s.write, s.buf1.write )
-    s.connect( s.read,  s.buf2.read  )
+    connect( s.write, s.buf1.write )
+    connect( s.read,  s.buf2.read  )
```

Comment:
How does the system know the different between `s.connect(s.q.out, s.q.in_)` and `s.q.connect(s.q.in_, s.q.out)`?

Reply:
I implement a way for connect to know which component calls the connect. https://github.com/cornell-brg/pymtl3/pull/71/files#diff-ee19c763e84ed97152d82ad3f94b778aR184